spyder/widgets/editor.py

```diff
@@ -1325,10 +1340,6 @@ def save_as(self, index=None):
 
             new_index = self.rename_in_data(index, new_filename=filename)
 
-            # We pass self object ID as a QString, because otherwise it would
-            # depend on the platform: long for 64bit, int for 32bit. Replacing
-            # by long all the time is not working on some 32bit platforms
-            # (see Issue 1094, Issue 1098)

```

Comment:
>Hi @goanpeca these comments do not belong to my amendment, so I preferred to remove trash.

If you did not made those comments, and the comments do not refer to code you are modifying, how is that trash??? please undo those changes. You should never remove comments of work that is not related to your work